api/python/cellxgene_census/tests/test_open.py

```diff
@@ -190,35 +190,6 @@ def test_open_soma_works_if_no_relative_uri_specified(requests_mock: rm.Mocker)
         )
 
 
-def test_open_soma_defaults_to_latest_if_missing_stable(requests_mock: rm.Mocker) -> None:
```

Comment:
no longer relevant, since this was testing the temporary case where we did not yet have a "stable" release in releases.json